src/Aspire.ProjectTemplates/templates/aspire-starter/9.1/Aspire-StarterApplication.1.Tests/Aspire-StarterApplication.1.Tests.csproj

```diff
@@ -17,11 +17,11 @@
     <PackageReference Include="Aspire.Hosting.Testing" Version="!!REPLACE_WITH_LATEST_VERSION!!" />
     <PackageReference Include="coverlet.collector" Version="6.0.2" Condition=" $(TestFramework) == 'xUnit.net' " />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.10.0" Condition=" $(TestFramework) != 'MSTest' " />
-    <PackageReference Include="MSTest" Version="3.4.3" Condition=" $(TestFramework) == 'MSTest' " />
+    <PackageReference Include="MSTest" Version="3.6.1" Condition=" $(TestFramework) == 'MSTest' " />
```

Comment:
This should align with whatever the latest version referenced by the MSTest templates in the .NET SDK 9.0.200 uses

Reply:
turns out that 3.6.4